### ISSUE TITLE
[Scrollspy nav] Cannot read property 'then' of undefined

### ISSUE THREAD

**Issue by Hrishikesh-K** [uikit/uikit#4382]
### UIkit Version

3.6.11

### Actual Behavior

Scrollspy nav doesn't scroll to target and throws error in console `Uncaught TypeError: Cannot read property 'then' of undefined at e.scrollTo, at e.click`

![image](https://user-images.githubusercontent.com/11403551/104902333-a0aaf880-59a4-11eb-9004-38b34c50b637.png)

### Expected Behavior

It should scroll normally.

### Reproduction Link

It's not always happening. I could not narrow it down to a single specific cause. Even on this page: https://600435926017e40007beb17c--hrishikeshk.netlify.app/projects/dropbox/, it doesn't happen on all links.  It happens on `3.4: Print ads`, `3.5: Television ads`, `3.6: Radio ads`, `3.8: SMS ads`, `3.9: Point Of Purchase ads` and `3.10: Out Of Home ads`. Also, once this error is triggered, the scrollspy nav doesn't even add the `active` class to the correct element.

### Steps to Reproduce the Problem

1. Visit: https://600435926017e40007beb17c--hrishikeshk.netlify.app/projects/dropbox/
2. Try clicking on any of the titles in the table of contents mentioned above.
3. Check console.

**Comment by janschoenherr** [comment]
It seems to work fine for me. Chrome Browser.

**Comment by Hrishikesh-K** [comment]
This is really strange. I had made sure many times many times that it wasn't a cache issue or something as I had faced the same issue back in some other version. Back then too it had fixed itself. So, I waited this time and opened this issue after facing it for like a week. The same happened in my local dev server and live, multiple devices and browsers (incognito too). In fact the screenshot I took was also from the time I just loaded the website in incognito. But, yeah you're right. The error doesn't occur anymore.

Sorry to bother sir.
